setup.py

```diff
@@ -21,10 +21,12 @@
     zip_safe=False,
     install_requires=[
         # -*- Extra requirements: -*-
+        "ckanapi"
```

Comment:
@tanderegg none of the unit tests import the file that depends on the api right now. Now that I'm more familiar with ckan, I think there's also a way for me to do this in native ckan without depending on the API.

For now, I'm just trying to exclude that from the code that's tested, since installing ckan locally (or on our automated test runners) is a bit annoying (you need to manually install their requirements.txt).